fe/fe-core/src/main/java/com/starrocks/sql/optimizer/operator/logical/LogicalIcebergScanOperator.java

```diff
@@ -53,6 +54,14 @@ public ScanOperatorPredicates getScanOperatorPredicates() {
         return this.predicates;
     }
 
+    public boolean hasUnknownColumn() {
```

Comment:
```suggestion
    public boolean hasUnknownColumnStats() {
```

Reply:
ok, just be same as hive, I'll modify them together.